xmake.lua

```diff
@@ -32,6 +32,14 @@ target("pscm") do
     add_files({
         "src/**.cpp",
         "$(buildir)/version.cpp"})
+    
+    if is_mode("coverage") then
+        add_cxxflags("-O0")
+        add_cxxflags("-fprofile-arcs")
+        add_cxxflags("-ftest-coverage")
```

Comment:
avoid multiple `add_cxxflags` invocation:
```suggestion
        add_cxxflags({
            "-O0",
            "-fprofile-arcs",
            "-ftest-coverage"})
```